src/FSharpLint.Console/Program.fs

```diff
@@ -79,6 +81,13 @@ let private start (arguments:ParseResults<ToolArgs>) (toolsPath:Ionide.ProjInfo.
         | Some _
         | None -> Output.StandardOutput() :> Output.IOutput
 
+    if arguments.Contains ToolArgs.Version then
+        let v = 
+            System.Reflection.Assembly.GetExecutingAssembly().GetCustomAttributes(false)
+            |> Seq.pick (fun a -> match a with | :? System.Reflection.AssemblyInformationalVersionAttribute as i -> Some i.InformationalVersion | _ -> None)
+        sprintf "Current version: %s" v |> output.WriteInfo
```

Comment:
@Thorium you're using System.Reflection ns twice, let's rather add an open clause at the beginning of the file?